_doc-build-windows/action.yml

```
@@ -308,10 +326,15 @@ runs:
       shell: powershell
       env:
         REQUIREMENTS_FILE: ${{ inputs.requirements-file }}
+        USE_UV: ${{ inputs.use-uv }}
       if: ${{ steps.check-requirements-file-exists.outputs.EXISTS_DOC_REQUIREMENTS == 'true' }}
       run: |
         .venv\Scripts\Activate.ps1
-        python -m pip install -r "$env:REQUIREMENTS_FILE"
+        if ($env:USE_UV -eq "true") {
+            uv pip install -r "$env:REQUIREMENTS_FILE"
+        } else {
+            python -m pip install -r "$env:REQUIREMENTS_FILE"
+        }
```

```suggestion
        if ($env:USE_UV -eq "true") {
          uv pip install -r "$env:REQUIREMENTS_FILE"
        } else {
          python -m pip install -r "$env:REQUIREMENTS_FILE"
        }
```
To match files formatting